src/redteam/providers/custom/index.ts

```diff
@@ -0,0 +1,922 @@
+import dedent from 'dedent';
```

Comment:
consider breaking it into a couple of files.  It's hard to grok. 

Does this work like the other multi-turn strategies?